assets/minecraft/font/default.json

```diff
@@ -1,5 +1,581 @@
 {
   "providers": [
+    {
```

Comment:
Could we find a more permanent way to keep these here? It's all gonna get overridden as soon as someone copy pastes the default.json the emotes plugin generates, not sure what the best solution is. Might be worth adding something to plugin itself if we can't split this up into another file?

Reply:
I'm thinking of moving the emotes to their own font file, which should let us have a separate provider file specifically for the emotes. However, this would prevent people from being able to copy and paste the character for the emote directly in chat, and force them to type the full string.

Reply:
Hmm, it prevents signs too. I think it's too much to give up honestly :(

Reply:
Ok, I've updated the build script and layout of the font directory so it's a bit more straightforward. Font files should now go in the corresponding folder, and will get combined to a single font file. That way, we can replace certain groups of providers without having to worry about accidentally overwriting other providers. I've also updated the build script so it doesn't throw errors when reading JSON files with UTF-8 characters.

Reply:
Ooh, I've considered this as well before though worry about what it means for people playing with the dev pack ~~aka mostly myself lol~~. I think we could make the script generate some sort of local cache in the form of a secondary resourcepack which could be applied on top of our current one. That should probably be enough, we could even skip some of the steps if the right files are present. What do you think about something like that?